src/pages/angular.mdx

```diff
@@ -229,10 +229,84 @@ export class AppComponent {
 }
 ```
 
-<a href="https://stackblitz.com/edit/swiper-angular-example?file=src/app/app.component.ts">
-  Edit on Stackblitz
+<a href="https://codesandbox.io/s/swiper-angular-example-9edj9">
+  Codesandbox playground with a lot of demos
 </a>
-;
+
+## Accessing swiper reference
+
+```js
+import { Component } from '@angular/core';
+
+// import Swiper core and required modules
+import SwiperCore, { Swiper } from 'swiper';
+
+// install Swiper modules
+SwiperCore.use([]);
+
+@Component({
+  selector: 'app-swiper-fade-example',
+  template: `
+    <swiper #swiper>
+      <ng-template swiperSlide>Slide 1</ng-template>
+      <ng-template swiperSlide>Slide 2</ng-template>
+      <ng-template swiperSlide>Slide 3</ng-template>
+      <ng-template swiperSlide>Slide 4</ng-template>
+      <ng-template swiperSlide>Slide 5</ng-template>
+      <ng-template swiperSlide>Slide 6</ng-template>
+      <ng-template swiperSlide>Slide 7</ng-template>
+    </swiper>
+    <button (click)="slideNext()">Next slide</button>
+    <button (click)="slidePrev()">Prev slide</button>
+  `,
+})
+export class AppComponent {
+  @ViewChild('swiper', { static: false }) swiper?: SwiperComponent;
+  slideNext(){
+    this.swiper.swiperRef.slideNext(100);
+  }
+  slidePrev(){
+    this.swiper.swiperRef.slidePrev(100);
+  }
+}
+```
+
+## Controlling slide index
```

Comment:
I am not sure about this part. We already have 2 ways of getting Swiper instance here: with `ref` and with `onSwiper` event. And with Swiper instance, user can call any api, e.g. `swiper.slideTo($index)`, which has some useful extra parameters.

I already foresee users question (and opened issues):

> What does setting this index do? Does it change slides with transition or not? If with transition, and I want to change it without transition, what should I do? 😄 

I would stick with unified single `swiper.slideTo` method to control this

Reply:
But this functionality exists from the release of swiper angular. just was undocumented :)
Do you want to remove this functionality from swiper angular too?

Reply:
Yes, we can remove it from swiper angular too :)